assets/blocks/jobs/index.jsx

```diff
@@ -0,0 +1,71 @@
+
+/**
+ * WordPress Dependencies.
+ */
+const { registerBlockType } = wp.blocks,
+	{ applyFilters } = wp.hooks;
+
+/**
+ * Internal Dependencies.
+ */
+import './style.scss';
+import edit from './edit.jsx';
+
+
+/**
+ * Base attributes configuration for the Jobs block. Other attributes may be
+ * added using the `wpjm_block_jobs_attributes_config` filter.
+ */
+const attributesConfig = {
+	showFilters: {
+		type: 'boolean',
+		default: true,
+	},
+	keywords: {
+		type: 'string',
+		default: '',
+	},
+	location: {
+		type: 'string',
+		default: '',
+	},
+};
+
+
+/**
+ * Register the Jobs block.
+ */
+registerBlockType( 'wp-job-manager/jobs', {
+	title: 'Jobs',
+	icon: 'list-view',
+	category: 'common',
+
+	attributes: applyFilters( 'wpjm_block_jobs_attributes_config', attributesConfig ),
+
+	edit,
+
+	save: function( { attributes } ) {
+		let shortcodeParams = {};
```

Comment:
It is generally more readable to use `const` here and the assign a new variable for the filter down below.

Reply:
Done